textile/fabric_printing/doctype/print_order/print_order.py

```diff
@@ -33,7 +33,7 @@
 )
 
 force_fields = force_customer_fields + force_fabric_fields + force_process_fields + force_process_component_fields
-
+ignore_permissions = frappe.has_permission("Print Order", "write")
```

Comment:
Very bad idea to keep a global variable here. Instead check permissions where ever necessary by `frappe.has_permission()` or `frappe.check_permission()`. And revert back to `ignore_permissions=True`